vcell-cli/src/main/java/org/vcell/cli/CLIUtils.java

```diff
@@ -39,6 +39,17 @@ private static void deleteRecursively(File f) throws IOException {
         }
     }
 
+    public static void cleanRootDir(File outdir){
+        // If this could be done without hard coding it'd be preferable.
```

Comment:
do you only want to remove these files or all files in the directory?  This is a somewhat unexpected side-effect.  It is one thing to overwrite without asking, it is another to remove files.

Reply:
as we discussed it may be good to remove only what is appended.